### ISSUE TITLE
Crash on save

### ISSUE THREAD

**Issue by ArdentParagon** [SkyrimTogether/issues-game#156]
## Description

When the game saves it freezes then crashes, saving worked before and suddenly it isn't anymore.

## Steps to reproduce

How to reproduce this issue:

1. Start the game.
2. Connect to the server.
3. Save (quick and normal save)

[20190127-155413-15640-20816.zip](https://github.com/SkyrimTogether/issues-game/files/2801374/20190127-155413-15640-20816.zip)

## Reproduction rate

I've done it 5 times in a row.

## Expected result

Not crashing?

## Your environment
 Riverwood and outside whiterun

* Game edition (choose on which edition do you have problems):
    * The Elder Scrolls V: Skyrim Special Edition

## Evidence (optional)

Show us what happen.

**Comment by f13rce** [comment]
5 times in a row on the same savegame? Or have you also tried a different character/older save? This may help me out to find some specifics.

**Comment by f13rce** [comment]
From what I can see in the dmp files, the crash is likely to be caused by something we spawned into the game. I'll have a look to see if I can find what this exactly is.

**Comment by f13rce** [comment]
Made some improvements and will be pushed live eventually. I'll let you know when it's life, so that you can confirm if the problem remains.

**Comment by F1r3w477** [comment]
@f13rce Last night, I had a friend who was having the same issue, saving would crash his game, he had to create a new character for it to stop.

**Comment by Serpent15** [comment]
Can confirm this issue, happened in both ST and launching it without, fixed by reverting to an older save.

**Comment by Skyblade799** [comment]
Yeah, just had this same issue happen a moment ago. Both me and my friend are having this occur. Once connected, saving crashes it;


[8efa5ce1-6cbf-4a81-bf21-574bdcda13d3.zip](https://github.com/SkyrimTogether/issues-game/files/2823920/8efa5ce1-6cbf-4a81-bf21-574bdcda13d3.zip)

**Comment by Inchs** [comment]
Same issue is occurring for me and friends, cannot save without crashing, this includes autosaving, quick saving and other saving.

**Comment by NexusR** [comment]
I find quick saving together with friends a couple of times will almost always reproduce the crash experienced here.

**Comment by F1r3w477** [comment]
A temporary fix that helps most people is to type player.kill in console once you load your save. Then it reloads the save and you should be able to save all you want.

**Comment by profesorrizo** [comment]
Experiencing same problem, using F1r3w477's temporary fix helps, but ST seems to de-sync and players stop seeing enemies/enemies are duplicated (x2 or sometimes x3).

[01bb581b-3545-46cd-9410-3dfd472fc9e0.zip](https://github.com/SkyrimTogether/issues-game/files/2848567/01bb581b-3545-46cd-9410-3dfd472fc9e0.zip)

**Comment by Darkalll** [comment]
i tried every fix on this thread but not a single one can help me , im still crashing when reloading the save after death

**Comment by ArcaneMusic** [comment]
Same Issue here. My save is before Dragon Stone, where I'm locked into the first part of Bleak Falls Barrow. Progressing out of the area, both to Skyrim and to the sanctum causes a crash, including hard and quick saves as well. Occurring for me now in both single player and in multiplayer lobbies.
